source/errors.rst

```diff
@@ -197,7 +197,7 @@ If multiple slugs are duplicated:
 
   {
     "status": 409,
-    "error": "Slugs already exist",
+    "error": "Slugs Already Exist",
```

Comment:
https://github.com/osuosl/timesync-node/blob/develop/src/errors.js#L169 shows the same string is used for single slugs that already exist and for multiple slugs that already exist.